### ISSUE TITLE
Summary of town banks in log during startup/shutdown

### ISSUE THREAD

**Issue by LlmDl** [TownyAdvanced/Towny#1341]
Originally reported on Google Code with ID 1314

```
I love towny logs, and it's great to see who has withdrawn and deposited money into
banks and such, it really helps as an admin.

However, I would love to be able to get summaries of bank totals, because sometimes,
wither it be due to changing economies, changing databases, bugs or data loss, I am
usually forced to restore money based on the word of our players since, depending on
the database used for the economy, there may not good ways to read the database.

In short, I humbly request an extension of Towny logging to include bank totals.

Thanks for listening. Keep up the good work. Love your plugin! ^_^
```

Reported by `Zombiemold` on 2012-09-05 20:43:41
